### PR TITLE
Add microsalt order validation skeleton

### PR DESCRIPTION
**Pull request by seallard**

### CODE REVIEW

File: cg/services/order_validation_service/workflows/microsalt/validation/field/utils.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from pydantic_core import ValidationError
+
+from cg.services.order_validation_service.models.errors import ValidationErrors
+
+
+def convert_errors(pydantic_errors: ValidationError) -> ValidationErrors:
```
Comment (seallard) [inline cg/services/order_validation_service/workflows/microsalt/validation/field/utils.py:6]: Not sure about this one, I think we should move the existing `convert_errors` up from the tomte module and extend it instead.

File: cg/services/order_validation_service/workflows/microsalt/models/sample.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+class MicroSaltSample(Sample):
+    control: ControlEnum | None = None
+    elution_buffer: str | None = None
+    extraction_method: str | None = None
+    organism: str | None = Field(default=None, max_length=32)
+    priority: PriorityEnum | None = None
+    quantity: int | None = None
+    reference_genome: str | None = Field(default=None, max_length=255)
+    sample_concentration: float | None = None
+    subject_id: str = Field(pattern=NAME_PATTERN, max_length=128)
```
Comment (islean) [inline cg/services/order_validation_service/workflows/microsalt/models/sample.py:20]: Remove subject_id. It can not be set in MicroSALT

**Review by islean — APPROVED** [review]
Looks good!

**Review by islean — APPROVED** [review]
Nice! Now we need to add some rules!

**Pull request merged**